packages/suite/src/views/dashboard/components/AssetsCard/index.tsx

```diff
@@ -70,6 +91,24 @@ const AssetsCard = () => {
     });
     const networks = Object.keys(assets);
 
+    const assetsData: assetType[] = networks
+        .map(symbol => {
```

Comment:
`filter` can be omitted by using `flatMap` instead of `map`. Returning an empty array will not add the value to the result. For example: `[0, 1, 2, 3, 4, 5].flatMap(n => n & 1 ? [] : n); // [0, 2, 4]`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

Reply:
Done, Thx for tip.